sdk/healthinsights/azure-health-insights-radiologyinsights/README.md

```diff
@@ -0,0 +1,122 @@
+[Product Documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- A [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+- An existing Cognitive Services Health Insights instance.
+
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-health-insights-radiologyinsights;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-health-insights-radiologyinsights</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authenticate the client
+
+In order to interact with the Health Insights Radiology Insights service, you'll need to create an instance of the RadiologyInsightsClient class.  You will need an **endpoint** and an **API key** to instantiate a client object.
+
+#### Get API Key
+
+You can obtain the endpoint and API key from the resource information in the [Azure Portal][azure_portal].
+
+Alternatively, you can use the [Azure CLI][azure_cli] snippet below to get the API key from the Health Insights resource.
+
+```bash
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+#### Create a RadiologyInsightsClient with an API Key Credential
+
+Once you have the value for the API key, you can pass it as a string into an instance of **AzureKeyCredential**. Use the key as the credential parameter
+to authenticate the client:
+
+```java com.azure.health.insights.radiologyinsights.buildasyncclient
+String endpoint = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_ENDPOINT");
+String apiKey = Configuration.getGlobalConfiguration().get("AZURE_HEALTH_INSIGHTS_API_KEY");
+
+RadiologyInsightsAsyncClient radiologyInsightsAsyncClient = new RadiologyInsightsClientBuilder()
+        .endpoint(endpoint).serviceVersion(RadiologyInsightsServiceVersion.getLatest())
+        .credential(new AzureKeyCredential(apiKey)).buildAsyncClient();
+```
+
+## Key concepts
+
+Radiology Insights currently supports one document from one patient. Please take a look [here][radiology_insights_inferences] for more detailed information about the inferences this service produces.  
+
+## Examples
+
+### Create a RadiologyInsights request and get the result using an asynchronous client
+
+Infer radiology insights from a patient's radiology report.
+
+```java com.azure.health.insights.radiologyinsights.inferradiologyinsights
+PollerFlux<PollOperationDetails, RadiologyInsightsInferenceResult> asyncPoller = radiologyInsightsAsyncClient
+        .beginInferRadiologyInsights(createRadiologyInsightsRequest());
+```
```

Comment:
nit, usually SDK would first give example code on sync client

(user not familiar with Reactor likely found it difficult to use async client)

Reply:
And as an example, the snippet would be better to include the code in `createRadiologyInsightsRequest`.

Else, reader probably have no idea how to create the `RadiologyInsightsData` object.

Reply:
Added both the sync and async example code to the readme file with precedence for sync call. The full request creation has also been included in the sample with last commit.